app/src/main/java/com/github/se/project/ui/profile/CreateUserProfile.kt

```diff
@@ -236,13 +237,6 @@ fun CreateProfileScreen(
                   }
 
                   return@Button
-                } catch (e: IllegalArgumentException) {
-                  Toast.makeText(
-                          context,
-                          "Please select a section and an academic level from the dropdown menu!",
-                          Toast.LENGTH_SHORT)
-                      .show()
```

Comment:
Because the way Alex modifed it is not a text input anymore, but a dropdown. So the error that would come out is "Please complete all the fields before creating your account!"

Reply:
Not really because yes we have a DropDownMenu but then for simplicity when we select a section (or a role, or an academic level), we convert the choice to a String and then at the end we convert it again to a Enum to create the Profile. So it will still need this exception catch. I will update it.